_data/contributors.js

```diff
@@ -3,81 +3,91 @@ const { AssetCache } = require('@11ty/eleventy-fetch');
 const { graphql } = require('@octokit/graphql');
 
 const ORGANIZATION_NAME = 'RedHat-Israel'; // organization namespace
-const MAX_CONTRIBUTORS = 100; // modify this if/when we have more then 100 members
+const MAX_CONTRIBUTORS_FETCH = 20;
```

Comment:
maybe this can also go in the env, so that it has a lower number for dev and higher for prod?

Reply:
@bennypowers 
we don't have a dev environment for gh api.

the limit for items per filter is 100, this was set to 20 because when higher, we risk gh internal timeout while fetching our items.
because of that high risk, and the fact that this is more of a system limitation that will rarely get modified, I'm not sure about exposing it as an environment variable.
WDYT?

Reply:
np